client/src/components/relatedProducts/relatedProductsList.jsx

```diff
@@ -71,37 +80,53 @@ const RelatedProducts = (props) => {
       })
   }, [product_id]);
 
+  // const handleStarClick =() => {
+  //   console.log('star is clicked from related')
+    // React.useEffect(()=> {
+      // setIsOpen(true)
+    // },[])
+
+  // }
+
   return (
     <List>
       <br />
       <br />
       <Carousel responsive={responsive}>
 
-        {accumulatedProductData.map((product, index) => {
+
+        {
+        accumulatedProductData.map((product, index) => {
           let style = product.styles.results[0]
 
           return (
             <Card
-
-            info={product.info}
+             style={style.style_id}
+             info={product.info}
               picUrls={style.photos.map(photo => photo.url)}
               category={product.info.category}
               name={product.info.name}
               price={'$' + style.original_price}
               salePrice={style.sale_price ?
-                '$' + style.sale_price : index % 2 === 0 ?
-                  '$7327.00' : null}
+                '$' + style.sale_price : null}
               key={index}
-              button={<StarBtn />}
+              // button={<FontAwesomeIcon  onClick={handleStarClick} style={second} icon={faStar} />}
+              btnStyle={'star'}
+              setIsOpen = {setIsOpen}
               handleChangeProduct ={props.handleChangeProduct}
```

Comment:
If it is in quotes I don't believe it needs to be in curly braces.

Reply:
roger, will correct that.